### ISSUE TITLE
v3.5.0.3 Incompatible with SPM 

### ISSUE THREAD

**Issue by Barbatosso** [AgoraIO/AgoraRtcEngine_iOS#6]
Hello!
We are still having an issue with updating Agora.io SDK because of 4 digits in versioning
![CleanShot 2021-09-22 at 16 21 20@2x](https://user-images.githubusercontent.com/18027418/134351465-f47564be-d92d-4eba-8698-6e4eec39283f.jpg)
![IMAGE 2021-09-22 16:22:17](https://user-images.githubusercontent.com/18027418/134351532-9807ed35-889c-4905-a423-de8a89df742c.jpg)
l

**Comment by maxxfrazer** [comment]
Hello, please refer to the semver compatible version in the releases:
https://github.com/AgoraIO/AgoraRtcEngine_iOS/releases/tag/3.5.0-r.3

version is `3.5.0-r.3`

**Comment by Rusik** [comment]
@maxxfrazer thanks for the answer, but we don't want to use exact version. We want to have "Up to Next Major" setting, but it is not works with this kind of versioning. Could you please just use standard SPM 3 digit versioning?

![Screenshot 2021-09-22 at 16 30 03](https://user-images.githubusercontent.com/800482/134352807-a602c670-c01d-4412-8e7d-e26733db65eb.png)

**Comment by maxxfrazer** [comment]
I agree, it's really annoying that our versioning causes these unnecessary complications.

I'll share this thread with the SDK team.
